README.md

```diff
@@ -7,16 +7,35 @@ A framework for integrating [UID2](https://github.com/IABTechLab/uid2docs) into
 
 ## Repository Structure
 
-TBD
+```
+.
+├── dev-app
+│   └── Development Application
+├── sdk
+│   ├── Sources
+│   └── Unit Tests
+├── securesignals-gma
+│   └── UID2 / GMA Plugin
+├── securesignals-ima
+│   └── UID2 / IMA Plugin
+├── build.gradle
+├── common.gradle
+├── LICENSE.md
+├── README.md
+```
 
 ## Requirements
 
-TBD
+* Android Studio (2022.1.1 Patch 2+)
+
+| Platform | Minimum target            |
+|----------|---------------------------|
+| Android  | 4.4+ (SDK) 5.0+ (Dev-App) |
```

Comment:
Let's add the actual API Version to this too for the SDK as KitKat supports both API 19 and API 20.

Reply:
Sounds good, done.